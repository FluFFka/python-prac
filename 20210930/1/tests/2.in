49,20,62,91,58,33,5,18,25,12,42,80,9,98,97,77,61,55,96,36,41,48,46,2,31,78,3,85,44,87,86,34,40,7,47,81,32,43,16,53,37,13,99,64,51,10,93,65,69,6,22,56,4,8,90,28,50,92,67,26,52,59,75,0,27,39,94,73,15,23,19,45,82,24,68,38,84,57,35,74,30,11,89,54,29,14,1,76,66,79,17,60,63,95,88,70,72,83,21,71
