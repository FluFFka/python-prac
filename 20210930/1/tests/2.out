[20, 80, 40, 10, 90, 50, 0, 30, 60, 70, 49, 99, 51, 1, 98, 48, 2, 52, 97, 3, 47, 53, 96, 46, 4, 54, 61, 39, 11, 89, 18, 32, 82, 68, 5, 25, 55, 85, 65, 75, 15, 45, 35, 95, 77, 27, 73, 23, 44, 6, 56, 94, 29, 79, 21, 71, 62, 12, 38, 88, 7, 43, 93, 57, 34, 16, 84, 66, 31, 81, 69, 19, 58, 42, 8, 92, 87, 37, 13, 63, 26, 24, 74, 76, 91, 9, 41, 59, 78, 22, 28, 72, 33, 67, 17, 83, 36, 86, 64, 14]
